README.md

```diff
@@ -761,6 +763,92 @@ context properties in web applications.
     }
     ```
 
+Microsoft.Extensions.Logging Integration
+----------------------------------------
+
+Provider notifies the Airbrake dashboard on an exception with the help of
+[Microsoft.Extensions.Logging](https://github.com/aspnet/Logging) methods.
+
+1. Install the `Sharpbrake.Extensions.Logging` package from NuGet (you can use "Package
+   Manager Console" from Visual Studio):
+
+   ```
+   PM> Install-Package Sharpbrake.Extensions.Logging
+   ```
+
+2. Configure the Airbrake provider through code:
+
+   2.1. In your `Startup.cs` add the import:
+
+   ```csharp
+   using Sharpbrake.Extensions.Logging
+   ```
+
+   2.2. In the `Configure` method (the same `Startup.cs` file) add the Airbrake provider to the logger factory:
+
+   ```csharp
+   loggerFactory.AddAirbrake(Configuration.GetSection("Airbrake"),
+       app.ApplicationServices.GetService<IHttpContextAccessor>());
+   ```
+
+   The code above assumes that the configuration options are defined in the registered configuration
+   file (e.g. `appsettings.json`). Example of such configuration file:
+
+   ```
+   {
+     "Logging": {
+       "IncludeScopes": false,
+       "LogLevel": {
+         "Default": "Warning"
+       }
+     },
+     "Airbrake": {
+       "ProjectId": "113743",
+       "ProjectKey": "81bbff95d52f8856c770bb39e827f3f6"
+     }
+   }
+   ```
+
+   To get access to the HTTP content `app.ApplicationServices.GetService<IHttpContextAccessor>()`
+   is used.
+
+   You can set the minimum log level for exceptions being sent to the dashboard
```

Comment:
I would write:

"Minimum log level can be set here to filter exceptions with lower severities (defaults to `LogLevel.Error`).